src/components/NcAppNavigation/NcAppNavigation.vue

```diff
@@ -161,6 +166,14 @@ export default {
 		 * @param {boolean} [state] set the state instead of inverting the current one
 		 */
 		toggleNavigation(state) {
+			// Early return if alreay in that state
+			if (this.open === state) {
+				emit('navigation-toggled', {
+					open: this.open,
+				})
+				return
+			}
```

Comment:
The behavior is not changes it just early returns, otherwise you have to wait for animation timeout.
Needed for users to know the state of the app navigation.

But it is no longer needed to be part of this PR (in the first version I waited for that event) but for e.g. screen readers it does not make any difference.